pom.xml

```diff
@@ -46,6 +47,7 @@
         <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-jar-plugin</artifactId>
+            <version>3.1.1</version>
```

Comment:
Can you add a variable for the version on top of the POM file? It makes maintenance easier.

Reply:
Sure.
- Added mavenjar.version as a variable